parsl/configs/osg.py

```diff
@@ -13,12 +13,17 @@
                 max_blocks=4,
                 # This scheduler option string ensures that the compute nodes provisioned
                 # will have modules
-                scheduler_options='Requirements = OSGVO_OS_STRING == "RHEL 6" && Arch == "X86_64" &&  HAS_MODULES == True',
+                scheduler_options="""
+                +ProjectName = "MyProject"
```

Comment:
Looks like the `+` sign is NOT accidentally added before `ProjectName`? https://support.opensciencegrid.org/support/solutions/articles/5000634360-join-and-use-a-project-in-osg-connect